tests/install_upgrade_operators/must_gather/utils.py

```diff
@@ -249,6 +249,14 @@ def check_logs(cnv_must_gather, running_hco_containers, namespace, label_selecto
 
 def compare_webhook_svc_contents(webhook_resources, cnv_must_gather, dyn_client, checks):
     for webhook_resource in webhook_resources:
+        # Skip webhooks with no configuration (e.g., Tekton dynamic webhooks)
+        if not webhook_resource.instance.webhooks:
```

Comment:
could this approach be hiding a bug where a webhok should have the config but does not have it?